### PR TITLE
Rust release 0.7.2

### PR DESCRIPTION
**Pull request by nazar-pc**
Changes:
* version bumps and changelog for new version
* readme is reused from the main one (I have modified it slightly)
* fixed examples due to `webpack-dev-server` having breaking changes in their beta versions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +27,7 @@ mediasoup and its client side libraries are designed to accomplish with the foll
 
 * Be a [SFU](https://webrtcglossary.com/sfu/) (Selective Forwarding Unit).
 * Support both WebRTC and plain RTP input and output.
-* Be a Node.js module in server side.
+* Be a Rust crate/Node.js module in server side.
```
Comment (ibc) [inline README.md]: Good try but, can we please put Node first at least for now? 😀

File: README.md [hunk 2]
```diff
@@ -51,15 +52,15 @@ mediasoup and its client side libraries provide a super low level API. They are
 
 ## Features
 
-* ECMAScript 6 low level API.
+* Idiomatic Rust/ECMAScript 6 low level API.
```
Comment (ibc) [inline README.md]: Same

File: rust/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+### 0.7.2
+* Thread and memory safety fixes in mediasoup-sys
```
Comment (ibc) [inline rust/CHANGELOG.md]:
```suggestion
### 0.7.2

* Thread and memory safety fixes in mediasoup-sys
```

Comment (ibc) [reply rust/CHANGELOG.md]: What happened to 0.7.1?

Comment (nazar-pc) [reply rust/CHANGELOG.md]: Nothing, quite literally. There was just a version bump due to documentation build errors.

**Review by ibc — APPROVED** [review]

**Comment by nazar-pc** [comment]
Published 0.7.2 to crates.io, please create a tag on master branch

**Comment by ibc** [comment]
`rust-0.7.2` tag pushed.

**Review by jmillan — COMMENTED** [review]
:+1:

**Pull request merged**